statsmodels/iolib/table.py

```diff
@@ -878,7 +878,8 @@ def set_datatype(self, val):
     stub=r'\textbf{%s}',
     empty='',
     missing='--',
-    replacements={"%" : "\%", ">" : "$>$", "|" : "$|$"}
+    #replacements will be processed in lexicographical order
```

Comment:
It looks like you added sorting to fix a unit test failure. Do you know what the problem was if the replacement sequence is arbitrary?